ansible_collections/arista/avd/roles/eos_validate_state/defaults/main.yml

```
@@ -19,6 +19,12 @@ fabric_name: "all"
 # Allow different manufacturers
 accepted_xcvr_manufacturers: "{{ validation_role.xcvr_own_manufacturers | arista.avd.default(['Arastra, Inc.', 'Arista Networks']) }}"
 
+# Allow different states for power supplies
+accepted_pwr_supply_states: "{{ validation_role.pwr_supply_states | arista.avd.default(['ok', 'Not Inserted']) }}"
+
+# Allow different states for fans
+accepted_fan_states: "{{ validation_role.fan_states | arista.avd.default(['ok', 'Not Inserted']) }}"
```

Update the defaults to OK only - the user must explicitly enable `Not Inserted``status.

```suggestion
accepted_fan_states: "{{ validation_role.fan_states | arista.avd.default(['ok']) }}"
```

Please also update README.md with a description of the use case.

---

Updated the main.yml.